daml-lf/data/src/main/scala/com/digitalasset/daml/lf/data/Time.scala

```diff
@@ -40,23 +38,15 @@ object Time {
     private val formatter: DateTimeFormatter =
       DateTimeFormatter.ISO_DATE.withZone(ZoneId.of("Z"))
 
-    private def assertDaysFromString(str: String) =
-      asInt(formatter.parse(str).getLong(ChronoField.EPOCH_DAY)).fold(sys.error, identity)
-
-    private[lf] def asInt(days: Long): Either[String, Int] = {
-      val daysI = days.toInt
-      if (daysI.toLong == days) Right(daysI)
-      else Left(s"out of bound Date $days")
-    }
```

Comment:
This safe version of `asInt` is useless here and I do not like how it is share with `SBuiltin`